packages/formation-react/src/components/AlertBox/usage.stories.mdx

```diff
@@ -20,63 +19,62 @@ Alerts keep users informed of important and sometimes time-sensitive changes.
 
 Used to provide helpful information to a user or something that warrants attention. Not used for negative consequences.
 
-<Story name="Info" args={{ ...defaultArgs }}>
-  {Template.bind({})}
-</Story>
+<Canvas>
+  <AlertBox {...defaultArgs} />
+</Canvas>
 
 ## Warning alert
 
 Used to warn a user, such as when there are negative consequences, but necessary when something has gone wrong.
 
-<Story name="Warning" args={{ ...defaultArgs, status: ALERT_TYPE.WARNING }}>
-  {Template.bind({})}
-</Story>
+<Canvas>
+  <AlertBox {...defaultArgs} status={ALERT_TYPE.WARNING} />
+</Canvas>
 
 ## Success alert
 
 Used to indicate success.
 
-<Story name="Success" args={{ ...defaultArgs, status: ALERT_TYPE.SUCCESS }}>
-  {Template.bind({})}
-</Story>
+<Canvas>
+  <AlertBox {...defaultArgs} status={ALERT_TYPE.SUCCESS} />
+</Canvas>
 
 ## Error alert
 
 Used when there is a problem or something destructive is about to occur.
 
 
-<Story name="Error" args={{ ...defaultArgs, status: ALERT_TYPE.ERROR }}>
-  {Template.bind({})}
-</Story>
+<Canvas>
+  <AlertBox {...defaultArgs} status={ALERT_TYPE.ERROR} />
+</Canvas>
 
 
 ## Sign in or tool prompt
 
 Used to prompt user to sign in, create an account, or launch a tool to access certain information.
 
 
-<Story name="Sign in" args={{
-  status: ALERT_TYPE.CONTINUE,
-  headline: 'You’ll need to sign in',
-  content: <div><p>Please sign in to continue.</p><button class="va-button-primary">Sign in or create an account</button></div>}}>
-  {Template.bind({})}
-</Story>
+<Canvas>
+  <AlertBox
+    {...defaultArgs}
```

Comment:
I don't think we need to spread `defaultArgs` since we explicitly set `status`, `headline`, and `content`.